index.js

```diff
@@ -1,11 +1,36 @@
 import AggregateError from 'aggregate-error';
 
+/**
+An error to be thrown when the request is aborted by AbortController.
+DOMException is thrown instead of this Error when DOMException is available.
+*/
+export class AbortError extends Error {
+	constructor(message) {
+		super();
+		this.name = 'AbortError';
+		this.message = message;
+	}
+}
+
+const getDOMException = errorMessage => globalThis.DOMException === undefined
+	? new AbortError(errorMessage)
+	: new DOMException(errorMessage);
+
```

Comment:
Can you add a TODO comment about what can be removed here when targeting Node.js 14 and 16.